{{cookiecutter.github_repository}}/Makefile

```diff
@@ -52,8 +52,11 @@ regenerate:  ## Delete and create new database.
 .PHONY: regenerate
 
 install: venv  ## Install and setup project dependencies
-	${ENV_PREFIX}python -m pip install --upgrade pip wheel
-	${ENV_PREFIX}python -m pip install -r requirements/development.txt
+	${ENV_PREFIX}python -m pip install --upgrade pip wheel poetry
```

Comment:
here we are installing `poetry` in venv, but I suggest we keep it outside as requirement for our cookiecutter. Also `poetry shell` mentioned elsewhere will only work if poetry is installed globally for that user. So either official poetry install or with `pipx`